### PR TITLE
Feature: cc-0041 update readme file

### PR DESCRIPTION
**Pull request by Jzeva**
Update the readme.md file in the front-end repo.

### CODE REVIEW

**Review by tatoMa — APPROVED** [review]
This is way better than before! Awesome work. Thank you.

File: README.md [hunk 1]
```diff
@@ -1,82 +1,58 @@
...
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
 You can start editing the page by modifying `src/pages/index.tsx`. The page auto-updates as you edit the file.
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `src/pages/api/hello.ts`.
-
 The `src/pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
 
-### Prepare First Commit
+### Environment Variables
```
Comment (kritomomo) [inline README.md:24]: Setup environment variables for local env

File: README.md [hunk 2]
```diff
@@ -1,82 +1,58 @@
...
 
-Add a file named .env at the root directory. Copy the JS from Frontend ENV ticket on notion(Get notion access permission from the development group) and paste it in .env.
+Add a file named `.env` at the root directory. Copy the .env code from Frontend ENV ticket on notion(Get notion access permission from the development group) and paste it in .env. You can refer to the `.env.example` file in the directory and repo.
```
Comment (kritomomo) [inline README.md:26]:
```
cp .env.example .env
```

**Review by kritomomo — APPROVED** [review]
LGTM

**Comment by EmilyGong111** [comment]
Wow, brilliant job. By the way, do we need to mention UAT testing environment  and database schema here? Because they are things our coworkers may need with their job.

**Comment by EmilyGong111** [comment]
Wow, brilliant job. Thanks for the tips. By the way, do we need to mention UAT testing environment and database schema here? Because they are things our coworkers may need with their work.

File: README.md (hunk 2 above)
Comment (Jzeva) [reply README.md:26]: I will fixing it now.

**Comment by Jzeva** [comment]
> Wow, brilliant job. Thanks for the tips. By the way, do we need to mention UAT testing environment and database schema here? Because they are things our coworkers may need with their work.

Personally I think these information should not be open to public. How do you think?

**Pull request merged**